src/form/acr.jl

```diff
@@ -87,6 +87,12 @@ function variable_mc_bus_voltage_on_off(pm::AbstractUnbalancedACRModel; nw::Int=
 end
 
 
+"Capacitor switching variables"
```

Comment:
should correct to follow new conventions for docstring format that include the function signature